FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_DNS.c

```diff
@@ -1249,22 +1705,16 @@ BaseType_t xReturn = pdTRUE;
 
 			if( ( pxDNSMessageHeader->usFlags & dnsRX_FLAGS_MASK ) == dnsEXPECTED_RX_FLAGS )
 			{
-			const uint16_t usCount = ( uint16_t ) ipconfigDNS_CACHE_ADDRESSES_PER_ENTRY;
-			uint16_t usNumARecordsStored = 0;
-	
-				for( x = 0U; x < pxDNSMessageHeader->usAnswers; x++ )
+				const uint16_t usCount = ( uint16_t ) ipconfigDNS_CACHE_ADDRESSES_PER_ENTRY;
+				uint16_t usNumARecordsStored = 0;
+
+				for( x = 0U; ( x < pxDNSMessageHeader->usAnswers ) && ( usNumARecordsStored < usCount ); x++ )
 				{
 				BaseType_t xDoAccept;
-	
-					if( usNumARecordsStored >= usCount )
-					{
-						/* Only count ipconfigDNS_CACHE_ADDRESSES_PER_ENTRY number of records. */
-						break;
-					}
 
 					uxResult = prvSkipNameField( pucByte,
 												 uxSourceBytesRemaining );
-	
+/*logging*/FreeRTOS_printf( ( "prvSkipNameField: %u  usAnswers = %u\n", uxResult, pxDNSMessageHeader->usAnswers ) );
```

Comment:
What is the use of `/*logging*/`.

Reply:
The same as above, will be taken away as soon as it is not necessary any more.